modules/administration/pages/image-management.adoc

```diff
@@ -1,12 +1,11 @@
 [[image-management]]
 = Image Building and Management
 
-
-
```

Comment:
I think we traditionally have 3 empty lines before every title of a section or procedure, etc.  In the following I do not comment on lost empty lines.